exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -867,7 +867,11 @@ func (b *BTCMarkets) GetOrderHistory(ctx context.Context, req *order.GetOrdersRe
 			tempResp.ID = tempData.Orders[c].OrderID
 			tempResp.Date = tempData.Orders[c].CreationTime
 			tempResp.Price = tempData.Orders[c].Price
+			tempResp.AverageExecutedPrice = tempData.Orders[c].Price
```

Comment:
Assume AverageExecutedPrice = Price as the data seems to be trades (from b.GetBatchTrades)